webapp/app.py

```diff
@@ -35,3 +59,14 @@ def internal_error(error):
 @app.context_processor
 def context():
     return dict(get_rss_feed=get_rss_feed)
+
+
+@app.route("/docs/api")
+def api():
+    """
+    Show the static api page
+    """
+
+    parser = DocParser(discourse_api, DOCS_TOPIC_ID, DOCS_URL_PREFIX)
```

Comment:
actually this was the reason why you had a variable sorry I missed this

Reply:
yeah but this was the thing we solved with version `0.6.0` of discourse-docs, so I'm just upgrading.